### PR TITLE
fix(appeals-service-api): fix helm lint issue with overly long config variable name

### PR DESCRIPTION
**Pull request by a6software**
## Ticket Number

https://pins-ds.atlassian.net/browse/AS-1864

## Description of change
fix helm lint issue with overly long config variable name

## Checklist

- [ ] Requires infrastructure changes
- [x] If adding or remove environment variables (e.g. in `docker-compose.yaml`) then I have updated the appropriate Helm chart
- [ ] I have updated the documentation accordingly
- [x] My commit history in this PR is linear
- [ ] New features have tests
- [ ] Breaking change (team conversation required)

## Important

Please do not merge from `master` (please only [rebase](https://github.com/foundry4/appeal-planning-decision/wiki/An-intro-to-Git-Rebase)). This keeps the history linear and easier to debug.

### CODE REVIEW

**Review by danWatkinson — APPROVED** [review]
Needs code owner approval to proceed, but putting my stamp on 👍

**Review by Russ-Clarke — APPROVED** [review]

**Comment by a6software** [comment]
thanks both

**Pull request merged**
